src/test/java/com/conveyal/gtfs/loader/JDBCTableWriterTest.java

```diff
@@ -421,6 +422,60 @@ public void canCreateUpdateAndDeleteScheduleExceptions() throws IOException, SQL
         assertThatSqlQueryYieldsZeroRows(getColumnsForId(scheduleException.id, scheduleExceptionTable));
     }
 
+    /**
+     * Ensure that {@link ScheduleException}s which are loaded from an existing GTFS can be removed properly,
+     * including created entries in calendar_dates.
+     */
+    @Test
+    public void canCreateAndDeleteCalendarDates() throws IOException, SQLException, InvalidNamespaceException {
+        String firstServiceId = "REMOVED";
+        String secondServiceId = "ADDED";
+        String[] allServiceIds = new String[]{firstServiceId, secondServiceId};
+        String[] holidayDates = new String[]{"20190812", "20190813", "20190814"};
```

Comment:
```suggestion
        String[] allServiceIds = new String[] {firstServiceId, secondServiceId};
        String[] holidayDates = new String[] {"20190812", "20190813", "20190814"};
```

Reply:
Updated in https://github.com/conveyal/gtfs-lib/pull/376/commits/577ae945553f3c5f6f2a480ae4a0588b5c40e116